docs/source/api_commands.rst

```diff
@@ -15,10 +15,37 @@ Parameters: None
 app_get_dryer_setting
 ---------------------
 
-Description:
+Description: Get docker dryer settings.
 
-Parameters:
+Parameters: None
+
+Returns:
+
+    status:
+
+    on:
+
+        cliff_on:
+
+        cliff_off
+
+        count:
+
+        dry_time: Duration dryer remains on in seconds.
 
+    off:
+    
+        cliff_on:
+
+        cliff_off:
+
+        count:
+
+Return example::
```

Comment:
Extra :

Reply:
This is to create a [literal block](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks) for the actual JSON response below.